utils/compiler/src/Command/DowngradePathsCommand.php

```diff
@@ -44,8 +44,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             'vendor/symfony/dependency-injection',
             'vendor/symfony/console',
             'vendor/symfony vendor/psr',
-            'vendor/symplify vendor/nikic vendor/ssch bin src packages rector.php',
+            'vendor/symplify vendor/nikic src packages',
+            'vendor/ssch',
```

Comment:
`vendor/ssch` is part that take longest time to downgrade, so split to separate process.